src/transformers/generation_utils.py

```diff
@@ -633,6 +647,7 @@ def generate(
         no_repeat_ngram_size: Optional[int] = None,
         encoder_no_repeat_ngram_size: Optional[int] = None,
         num_return_sequences: Optional[int] = None,
+        max_time: Optional[float] = None,
```

Comment:
Can we let `max_time` default to `None` in `configuration_utils.py`?

Reply:
Do you really think we should add this, to the configuration files ?
Generation time seems like something caller dependent, not really model dependent. (I mean why would a model limit it's generation time by default to X seconds? The results would depend on GPU/CPU exact hardware etc...)

Reply:
I can really add it, but it feels odd

Reply:
Yeah, I agree with you here! Let's not add it to the config for now - the general philosophy for generate used to be to just add everything to the config, but as you pointed out, it really doesn't make much sense for config params like `max_time`